### PR TITLE
Move node token to /etc/tokens/node

### PR DESCRIPTION
**Pull request by r7vme**
part of https://github.com/giantswarm/giantswarm/issues/2642

### CODE REVIEW

File: cloud-config/master.yaml.tmpl [hunk 1]
```diff
@@ -1893,7 +1900,7 @@ coreos:
       quay.io/giantswarm/certctl:b07d0913d5cb369a6b605394bdd4be4633451be9 \
       issue \
       --vault-addr=https://${VAULT_DOMAIN_NAME} \
-      --vault-token=${G8S_VAULT_TOKEN} \
+      --vault-token=$${VAULT_TOKEN} \
```
Comment (corest) [inline cloud-config/master.yaml.tmpl]: `$$` -> $`

File: cloud-config/master.yaml.tmpl [hunk 2]
```diff
@@ -1925,7 +1933,7 @@ coreos:
...
```
Comment (corest) [inline cloud-config/master.yaml.tmpl]: `$$` -> `$` ?

File: cloud-config/master.yaml.tmpl [hunk 3]
```diff
@@ -2171,8 +2179,8 @@ coreos:
       -v /etc/kubernetes/ssl/:/etc/kubernetes/ssl/ \
       quay.io/giantswarm/certctl:b07d0913d5cb369a6b605394bdd4be4633451be9 \
       issue \
-      --vault-addr=https://${VAULT_DOMAIN_NAME} \
-      --vault-token=${G8S_VAULT_TOKEN} \
+      --vault-addr=$${VAULT_ADDR} \
```
Comment (corest) [inline cloud-config/master.yaml.tmpl]: `$$` -> `$` ?

File: cloud-config/master.yaml.tmpl [hunk 4]
```diff
@@ -2171,8 +2179,8 @@ coreos:
...
+      --vault-token=$${VAULT_TOKEN} \
```
Comment (corest) [inline cloud-config/master.yaml.tmpl]: `$$` -> `$` ?

Comment (r7vme) [reply cloud-config/master.yaml.tmpl]: Noop, it's intentionally, See here https://www.terraform.io/docs/providers/template/d/file.html#inline-templates

`$$` used to escape `$`

Comment (corest) [reply cloud-config/master.yaml.tmpl]: Ok, my bad
lgtm then

**Pull request merged**